libs/infinity_emb/infinity_emb/transformer/vision/torch_vision.py

```diff
@@ -18,49 +27,106 @@
 if CHECK_TORCH.is_available:
     import torch
 if CHECK_TRANSFORMERS.is_available:
-    from transformers import AutoModel, AutoProcessor  # type: ignore
+    from transformers import AutoConfig, AutoModel, AutoProcessor  # type: ignore
+if CHECK_PIL.is_available:
+    from PIL import Image
 
 
-class ClipLikeModel(BaseClipVisionModel):
+class TorchImageModel(BaseClipVisionModel):
```

Comment:
For consistency, wouldn't make sense to also rename the base class here?

Reply:
Resolved!